src/query/interpret/eval.hpp

```diff
@@ -837,6 +838,7 @@ class ExpressionEvaluator : public ExpressionVisitor<TypedValue> {
 
   TypedValue Visit(Function &function) override {
     FunctionContext function_ctx{dba_, ctx_->memory, ctx_->timestamp, &ctx_->counters, view_};
+    bool is_transactional = MgpGraphIsTransactional(mgp_graph::NonWritableGraph(*dba_, view_));
```

Comment:
I think we should actually call dba_ to get us info whether we are in transactional state or not. So, idea is to have function which dba_ can call to return isTransactional, without calling mg procedure impl

Reply:
Moving the implementaton to the accessor also makes sense to me :+1: